model-linter/src/test/java/org/ndx/aadarchi/model/linter/MemoryAppender.java

```diff
@@ -0,0 +1,41 @@
+package org.ndx.aadarchi.model.linter;
+
+
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+
+import java.util.List;
+import ch.qos.logback.classic.Level;
+import java.util.stream.Collectors;
+
+public class MemoryAppender extends ListAppender<ILoggingEvent> {
```

Comment:
Can you convert this to directly use the java.util.logging appender API ? (and maybe extract it to some test util module, in order for it to be usable elsewhere)